File: src/main/scala/analysis/session/bean/SessionRandomExtract.scala
package analysis.session.bean

/**
* Session随机抽取表
*
* @param taskid             当前计算批次的ID
* @param sessionid          抽取的Session的ID
* @param startTime          Session的开始时间
* @param searchKeywords     Session的查询字段
* @param clickCategoryIds   Session点击的类别id集合
*/
case class SessionRandomExtract(taskid:String,
                                sessionid:String,
                                startTime:String,
                                searchKeywords:String,
                                clickCategoryIds:String)

File: src/main/java/com/jxust/entity/SessionRandomExtract.java
package com.jxust.entity;

import java.io.Serializable;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;


/**
* Create by Jayden in 2020/3/7
*/

@Data
@AllArgsConstructor
@NoArgsConstructor
public class SessionRandomExtract implements Serializable {
    private String taskid;

    private String sessionid;

    private String starttime;

    private String searchkeywords;

    private String clickcategoryids;

    private static final long serialVersionUID = 1L;
}